src/charts/candle/useDatetime.ts

```diff
@@ -30,13 +30,14 @@ export function useCandlestickChartDatetime({
 
   const formatted = useDerivedValue(() => {
     if (timestamp.value === -1) return '';
+    let index = 0
```

Comment:
Since we don't know the `index` here it's probably better if we provide a `null` or `undefined` value. This would also require a change to the `TFormatterFn` type. What do you think?